x/auth/ante/ante_test.go

```diff
@@ -1121,62 +1102,6 @@ func TestAnteHandlerSetPubKey(t *testing.T) {
 			false,
 			sdkerrors.ErrWrongSequence,
 		},
-		{
-			"make sure previous public key has been set after wrong signature",
```

Comment:
this test is not valid anymore because each account has it's own set pub key step